### DISCUSSION TITLE
Using build-time theming with NextJS

### DISCUSSION THREAD

**Discussion by Bergmam** [cloudscape-design/components#1504] [Q&A] [answered]
Hi, 

I am trying to apply build-time theming in my NextJS project, but after generating my `themed` folder as such:

```
buildThemedComponents({
  theme,
  outputDir: join(__dirname, './themed'),
})
```

I run into the following error when trying to import the components from `./theme/themed/components`:

```
./theme/themed/components/alert/actions-wrapper/styles.scoped.css
Global CSS cannot be imported from files other than your Custom <App>. Due to the Global nature of stylesheets, and to avoid conflicts, Please move all first-party global CSS imports to pages/_app.js. Or convert the import to Component-Level CSS (CSS Modules).
Read more: https://nextjs.org/docs/messages/css-global
Location: theme/themed/components/alert/actions-wrapper/styles.css.js
```

How would I go about allowing NextJS to include the themed component css files? 

I have been following the NextJS-specific instructions here: https://cloudscape.design/get-started/integration/using-cloudscape-components/
as well as followed the previous discussion here: https://github.com/cloudscape-design/components/discussions/954

**Comment by just-boris** [comment, marked as answer]
Seems related to this issue in Next: https://github.com/vercel/next.js/issues/39516

Please follow up with Next community, it is outside our capabilities to work around their limitations
